multi-node/vagrant/Vagrantfile

```diff
@@ -161,6 +162,9 @@ Vagrant.configure("2") do |config|
 
       controller.vm.provision :file, :source => CONTROLLER_CLOUD_CONFIG_PATH, :destination => "/tmp/vagrantfile-user-data"
       controller.vm.provision :shell, :inline => "mv /tmp/vagrantfile-user-data /var/lib/coreos-vagrant/", :privileged => true
+
+      controller.vm.provision :file, :source => CALICO_MANIFEST, :destination => "/tmp/calico.yaml"
+      controller.vm.provision :shell, :inline => "mkdir -p /srv/kubernetes/manifests && mv /tmp/calico.yaml /srv/kubernetes/manifests/", :privileged => true
```

Comment:
I was initially trying to avoid putting it into the script, but if that would be preferred I don't see a problem with that.